projects/batfish-client/src/main/java/org/batfish/client/BfCoordWorkHelper.java

```diff
@@ -152,22 +157,24 @@ public boolean delAnalysis(String containerName, String analysisName) {
 
   public boolean delContainer(String containerName) {
     try {
-      WebTarget webTarget = getTarget(CoordConsts.SVC_RSC_DEL_CONTAINER);
+      WebTarget webTarget =
+          getTargetV2(Lists.newArrayList(CoordConsts.SVC_KEY_CONTAINERS, containerName));
```

Comment:
this should be `container/name`, not `containers/name`.

Reply:
I think Github is using plural form for all resource nouns. [get-single-issue](https://developer.github.com/v3/issues/#get-a-single-issue), [get-single-PR](https://developer.github.com/v3/pulls/#get-a-single-pull-request), [delete-a-comment](https://developer.github.com/v3/issues/comments/#delete-a-comment)